tasks/src/sentence-similarity/data.ts

```diff
@@ -66,25 +66,25 @@ const taskData: TaskData = {
 	libraries: TASKS_MODEL_LIBRARIES["sentence-similarity"],
 	metrics:   [
 		{
-			description: "The reciprocal rank is a measure used to rank the relevancy of documents given a set of documents. Reciprocal Rank is the reciprocal of the rank of the document retrieved, meaning, if the rank is 3, the Reciprocal Rank is 0.33. If the rank is 1, the Reciprocal Rank is 1.",
+			description: "Reciprocal ranking is a measure used to rank the relevance of documents given a set of documents. The Reciprocal Rank is the reciprocal of the retrieved document rank, that is, if the rank is 3, the Reciprocal Rank is 0.33. If the rank is 1, the Reciprocal Rank is 1.",
 			id:          "Mean Reciprocal Rank",
 		},
 		{
-			description: "The similarity of embeddings is evaluated mainly on cosine similarity. It’s calculated as the cosine of the angle between two vectors. It is particularly useful when your documents do not have the same length.",
+			description: "The similarity of the embeddings is evaluated mainly based on the similarity of the cosine. It is calculated as the cosine of the angle between two vectors. It is particularly useful when your texts are not the same length.",
 			id:          "Cosine Similarity",
 		},
 	],
 	models: [
 		{
-			description: "This is a good model that works for sentences and paragraphs and can be used for clustering and semantic search.",
```

Comment:
Clustering is a term so I wouldn't change it.

Reply:
I actually think grouping might be ok since for our audience "clustering" might not make sense

Reply:
Perfect. I think I can apply both with 'clustering/grouping' :)